src/VirtoCommerce.CustomerExportImportModule.Data/Services/CustomerExportPagedDataSourceFactory.cs

```diff
@@ -0,0 +1,24 @@
+using VirtoCommerce.CustomerExportImportModule.Core.Models;
+using VirtoCommerce.CustomerExportImportModule.Core.Services;
+using VirtoCommerce.CustomerModule.Core.Services;
+
+namespace VirtoCommerce.CustomerExportImportModule.Data.Services
+{
+    public sealed class CustomerExportPagedDataSourceFactory: ICustomerExportPagedDataSourceFactory
+    {
+        private readonly IMemberService _memberService;
+        private readonly IMemberSearchService _memberSearchService;
+
```

Comment:
Only one line should separate fields and constructor

Reply:
Fixed